agent/acs/handler/acs_handler.go

```diff
@@ -372,7 +372,11 @@ func (acsSession *session) startACSSession(client wsclient.ClientServer) error {
 			// Stop receiving and sending messages from and to ACS when
 			// client.Serve returns an error. This can happen when the
 			// the connection is closed by ACS or the agent
-			seelog.Infof("ACS connection closed: %v", err)
+			if err == nil || err == io.EOF {
```

Comment:
for consistency with tcs: https://github.com/aws/amazon-ecs-agent/blob/master/agent/tcs/handler/handler.go#L83